Description: fix proto import paths, this dir layout is different
Forwarded: not-needed
--- android-platform-frameworks-base-10.0.0+r36.orig/tools/aapt2/Resources.proto
+++ android-platform-frameworks-base-10.0.0+r36/tools/aapt2/Resources.proto
@@ -16,7 +16,7 @@
 
 syntax = "proto3";
 
-import "frameworks/base/tools/aapt2/Configuration.proto";
+import "Configuration.proto";
 
 package aapt.pb;
 
--- android-platform-frameworks-base-10.0.0+r36.orig/tools/aapt2/ResourcesInternal.proto
+++ android-platform-frameworks-base-10.0.0+r36/tools/aapt2/ResourcesInternal.proto
@@ -16,8 +16,8 @@
 
 syntax = "proto3";
 
-import "frameworks/base/tools/aapt2/Configuration.proto";
-import "frameworks/base/tools/aapt2/Resources.proto";
+import "Configuration.proto";
+import "Resources.proto";
 
 package aapt.pb.internal;
 
